Automattic/jetpack
# Blocks: use Jetpack green on all block icons

#### Changes proposed in this Pull Request:

* Color all icons on Jetpack blocks with the green color.

#### Before

![image](https://user-images.githubusercontent.com/390760/78571371-023b1580-781e-11ea-86f7-25aedf80b2ab.png)


#### After

![image](https://user-images.githubusercontent.com/390760/78571355-ff402500-781d-11ea-8b2a-73d9ed98271f.png)


#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* No.

#### Testing instructions:

* Fire up this PR.
* Create a new post/page.
* Add a new Jetpack block either by going to the JP section or searching for one.
* Ensure all the blocks have the green colored icon.

#### Proposed changelog entry for your changes:
* None.
